src/Game/Managers/AudioManager.cs

```diff
@@ -158,6 +192,16 @@ public void StopMusic()
             }
         }
 
+        public void StopSounds()
+        {
+            foreach (UOSound sound in _currentSounds.ToArray())
```

Comment:
I think you can convert this code to a simple for.

```csharp
for (int i = 0; i < _currentSounds.Count; i++)
{
   var s = _currentSounds[i];
// do stuff
_currentSounds.RemoveAt(i--);
}
```

Reply:
Done